document/core/syntax/conventions.rst

```diff
@@ -37,8 +37,9 @@ The following conventions are adopted in defining grammar rules for abstract syn
 
 * Productions are written :math:`\X{sym} ::= A_1 ~|~ \dots ~|~ A_n`.
 
-* Some productions are augmented with side conditions in parentheses, ":math:`(\iff \X{condition})`", that provide a shorthand for a combinatorial expansion of the production into many separate cases.
+* Large productions may be split into multiple definitions, indicated by ending the first one with explicit ellipses, :math:`\X{sym} ::= A_1 ~|~ \dots`, and starting continuations with ellipses, :math:`\X{sym} ::= \dots ~|~ A_2`.
 
+* Some productions are augmented with side conditions in parentheses, ":math:`(\iff \X{condition})`", that provide a shorthand for a combinatorial expansion of the production into many separate cases.
```

Comment:
One other suggestion I missed before; I think "a shorthand to allow a combinatorial expansion..." would more clearly indicate that the notation isn't itself shorthand for the expansion, but its use allows the expanson.

Reply:
Hm, not sure I understand. What is the actual shorthand if not the use of the notation itself?